app/views/teams/_form.html.erb

```diff
@@ -4,7 +4,7 @@
   <%= render partial: "application/validation_errors", locals: {model: @team} %>
 
   <div class="form-group">
-    <%= f.label t("helpers.name"), :class => 'control-label col-lg-2' %>
+    <%= f.label t("helpers.name"), :class => 'control-label col-lg-2 required' %>
     <div class="col-lg-10">
       <%= f.text_field :name, :class => 'form-control' %>
```

Comment:
You could add `:required => true` to this (required) text field in order to validate the field content browser-side on supporting browsers.

The result would look like this when trying to submit the form with an empty required field:
<img width="490" alt="bildschirmfoto 2018-01-10 um 22 30 12" src="https://user-images.githubusercontent.com/807665/34796455-ddb725b0-f655-11e7-9332-20c16c8b19d3.png">

Reply:
did it, but i am still using the class i added to reference the field in out stylesheet in order to avoid modifying other required fields, okay?